OpenVPNChild.cs

```diff
@@ -27,43 +29,14 @@ public OpenVpnChild(OpenVpnServiceConfiguration config, string configFile)
              * so make sure we are working on a copy */
             this.configFile = String.Copy(configFile);
             this.exitEvent = Path.GetFileName(configFile) + "_" + Process.GetCurrentProcess().Id.ToString();
-            var justFilename = System.IO.Path.GetFileName(configFile);
-            var logFilename = config.logDir + "\\" +
-                    justFilename.Substring(0, justFilename.Length - config.configExt.Length) + ".log";
+            var justFilename = Path.GetFileName(configFile);
+            logFile = Path.Combine(config.logDir, justFilename.Substring(0, justFilename.Length - config.configExt.Length) + ".log");
 
             // FIXME: if (!init_security_attributes_allow_all (&sa))
```

Comment:
What's this?

Reply:
It appears to be a leftover from a previous C-based implementation (?). This commit just splits code into multiple files without any functional changes.

Reply:
Why not remove it if it's not needed anymore, this is the time.

Reply:
Well, hard to disagree. Removed!